robot9330.java

```diff
@@ -231,213 +285,204 @@ public void moveForwardOrReverse(int ticks, double power, int direction_id) {
             }
-            
             if (atonnymus_motorDriveBackRight_reverse == false) {
                 motorDriveBackRight.setTargetPosition(ticks);
                 motorDriveBackRight.setPower(power);
             } else {
                 motorDriveBackRight.setTargetPosition(-ticks);
                 motorDriveBackRight.setPower(-power);
             }
-            
             motorDriveFrontLeft.setMode(DcMotor.RunMode.RUN_TO_POSITION);
             motorDriveFrontRight.setMode(DcMotor.RunMode.RUN_TO_POSITION);
             motorDriveBackLeft.setMode(DcMotor.RunMode.RUN_TO_POSITION);
             motorDriveBackRight.setMode(DcMotor.RunMode.RUN_TO_POSITION);
         }
-        
+
     }
-    
-    //Wait for "motorDriveFrontRight" (Front right motor) to reach its encoder before entering the next atonnymus stage.
+
+    // Wait for "motorDriveFrontRight" (Front right motor) to reach its encoder
+    // before entering the next atonnymus stage.
     public void waitForMotorEncoderToFinnish() {
-        while(motorDriveFrontRight.isBusy()) {
-            
+        while (motorDriveFrontRight.isBusy()) {
+
         }
     }
-    
+
     public void autoBB() {
-        
-                                                    //Distance \ power \ direction.
+
+        // Distance \ power \ direction.
         moveForwardOrReverse(toTicks_ForwardAndBack(autoForward), autoPower, 0);
         waitForMotorEncoderToFinnish();
         moveForwardOrReverse(toTicks_ForwardAndBack(autoBackward), autoPower, 1);
         waitForMotorEncoderToFinnish();
         strafe(toTicks_LeftAndRight(autoBackdrop), autoPower, 0);
         waitForMotorEncoderToFinnish();
-        
-        
     }
-    
-    
+
     public void autoBW() {
-        
         moveForwardOrReverse(toTicks_ForwardAndBack(autoForward), autoPower, 0);
         waitForMotorEncoderToFinnish();
         moveForwardOrReverse(toTicks_ForwardAndBack(autoBackward), autoPower, 1);
         waitForMotorEncoderToFinnish();
         strafe(toTicks_LeftAndRight(autoWing), autoPower, 0);
         waitForMotorEncoderToFinnish();
     }
-    
-    
-    
+
     public void autoRB() {
-        
         moveForwardOrReverse(toTicks_ForwardAndBack(autoForward), autoPower, 0);
         waitForMotorEncoderToFinnish();
         moveForwardOrReverse(toTicks_ForwardAndBack(autoBackward), autoPower, 1);
         waitForMotorEncoderToFinnish();
         strafe(toTicks_LeftAndRight(autoBackdrop), autoPower, 1);
         waitForMotorEncoderToFinnish();
     }
-    
-    
+
     public void autoRW() {
-        
         moveForwardOrReverse(toTicks_ForwardAndBack(autoForward), autoPower, 0);
         waitForMotorEncoderToFinnish();
         moveForwardOrReverse(toTicks_ForwardAndBack(autoBackward), autoPower, 1);
         waitForMotorEncoderToFinnish();
         strafe(toTicks_LeftAndRight(autoWing), autoPower, 1);
         waitForMotorEncoderToFinnish();
-        
     }
 }
 
+/*
+ * Plane launcher notes:
```

Comment:
All of this vestigial code can be removed. Git keep a complete history of the entire project, so we always have this information. It's a bit redundant. Harmless at best, could mess us up at worst by being inaccurate.